modules/container/src/main/java/org/mule/runtime/container/internal/MuleClassLoaderLookupPolicy.java

```diff
@@ -98,21 +99,15 @@ public LookupStrategy getClassLookupStrategy(String className) {
 
   @Override
   public LookupStrategy getPackageLookupStrategy(String packageName) {
-    LookupStrategy lookupStrategy = lookupStrategies.get(packageName);
```

Comment:
beware, this is not initializing the map, but rather populating it. Don't think we need to do any change here for tis class.